stories/src/main/java/com/wordpress/stories/compose/story/StoryFrameItem.kt

```diff
@@ -89,5 +89,10 @@ data class StoryFrameItem(
                     }
             )
         }
+
+        @JvmStatic
+        fun getAltTextFromFrameAddedViews(frame: StoryFrameItem): String {
+            return frame.addedViews.joinToString(separator = " ") { it -> "${it.viewInfo.addedViewTextInfo.text}" }
```

Comment:
This can be simplified (unnecessary string template, and the arrow isn't needed either):

```suggestion
            return frame.addedViews.joinToString(separator = " ") { it.viewInfo.addedViewTextInfo.text }
```

Reply:
Since this change is pretty innocent and we'd like to get this change into the current beta, I'll just apply it and merge, forgive me Mario 🙏

Reply:
thank you! that was a nice one 👍